ui/app/styles/utils/_size_variables.scss

```diff
@@ -3,27 +3,31 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-/* General sizing in rem values used largely for text sizing.*/
-$size-1: 3rem; // 48px, same as $spacing-xxl
-$size-2: 2.5rem; // 40px
-$size-3: calc(24 / 14) + 0rem; // ~1.714rem ~27px
-$size-4: 1.5rem; // 24px same as $spacing-l
-$size-5: 1.25rem; // 20px
-$size-6: 1rem; // 16px same as $spacing-m
-$size-7: calc(13 / 14) + 0rem; // ~.929rem ~15px
-$size-8: calc(12 / 14) + 0rem; // ~.857rem  ~13.7px
-$size-9: 0.75rem; // 12px same as $spacing-s
-$size-10: 0.5rem; // 8px same as $spacing-xs
-$size-11: 0.25rem; // 4px same as spacing-xxs
+/* Size variables (rem only) */
```

Comment:
This table maps rem to pixel value using a base font size of 14pixels so spacing changes remain consistent when the base font size is increased 

| size var | rem value            | pixel value |
| -------- | -------------------- | ----------- |
| $size-1  | 3rem                 | 42px,       |
| $size-2  | 2.5rem               | 35px        |
| $size-3  | calc(24 / 14) + 0rem | 24px       |
| $size-4  | 1.5rem               | 21px        |
| $size-5  | 1.25rem              | 17.5px      |
| $size-6  | 1rem                 | 14px        |
| $size-7  | calc(13 / 14) + 0rem | 13px        |
| $size-8  | calc(12 / 14) + 0rem | 12px        |
| $size-9  | 0.75rem              | 10.5px      |
| $size-10 | 0.5rem               | 7px         |
| $size-11 | 0.25rem              | 3.5px       |